src/app/containers/Header/Logo.jsx

```diff
@@ -53,6 +54,14 @@ export default ({
             >
                 {settings.productName}
             </div>
+            <div
```

Comment:
This was a relic from when I wanted to have swordfish and gladius update notifications in the same place